src/vm/i386/excepx86.cpp

```diff
@@ -3769,10 +3783,27 @@ AdjustContextForVirtualStub(
     return TRUE;
 }
 
+#ifndef DACCESS_COMPILE
+#ifndef WIN64EXCEPTIONS
 #ifdef FEATURE_PAL
+EXTERN_C
+_Unwind_Reason_Code
+UnhandledExceptionHandlerUnix(
```

Comment:
And, we need to embrace ``UnhandledExceptionHandlerUnix`` with ``WIN64EXCEPTIONS`` in order to remove duplicated definition if we decide to leave it in ``unixstubs.cpp``.